joomla/joomla-cms
# [New Feature] Show the TFA Status in the Backend Users view

### Summary of Changes

After this PR we show if the user has enabled or disabled the TFA
### Testing Instructions
- Apply this patch
- Check the Users view in the backend
- see the new colum `TFA Status` is disabled
- enabled tfa for an user
- see that it shows that you have TFA enabled.
### Documentation Changes Required

Maybe the help screens?
